docs/lorawan-on-helium/frequency-plans/frequency-plans.mdx

```diff
@@ -0,0 +1,429 @@
+---
+id: frequency-plans
+hide_title: true
+sidebar_label: Frequency Plans
+slug: /lorawan-on-helium/frequency-plans
+---
+
+## Frequencies on the Helium Network
+
+Helium's LoRaWAN Network is built upon 8-channel gateways. This page will help
+you confirm the appropriate frequencies for gateway configuration each country.
```

Comment:
Needs an extra `for` after configuration.

Reply:
Good catch, thanks.